sycl/test-e2e/Matrix/joint_matrix_hip_gfx90a.cpp

```diff
@@ -6,10 +6,10 @@
 //
 //===----------------------------------------------------------------------===//
 
-// RUN: %{build} -fsycl -fsycl-targets=amd_gpu_gfx90a %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=amd_gpu_gfx90a %s -o %t.out
 // RUN: %{run} %t.out
 
-// REQUIRES: gpu-amd-gfx90a
+// REQUIRES: arch-amd_gpu_gfx90a
```

Comment:
this change is just to remove a hack in the testing infra, should be exactly the same functionally

Reply:
hey, just wondering if you were able to build and run these gfx90a tests. AFAIK we don't have these in our CI, so I don't think they ever ran.

Reply:
yeah i actually tested on the one AMD machine i know not in our CI and it happens to be gfx90a (MI250X)